chore: introduce dev environment setup for documentation (#2609)
keptn/lifecycle-toolkit: docs-new/CONTRIBUTING.md

# Contribute to the Keptn documentation

This document provides information about contributing to
the [Keptn documentation](https://lifecycle.keptn.sh/docs/),
which is part of the [Keptn](https://keptn.sh) website.

The Keptn documentation is authored with
[markdown](https://www.markdownguide.org/basic-syntax/)
and rendered using
[MkDocs](https://www.mkdocs.org/).

We welcome and encourage contributions of all levels.
You can make modifications using the GitHub editor;
this works well for small modifications but,
if you are making significant changes,
you may find it better to fork and clone the repository
and make changes using the text editor or IDE of your choice.
You can also run the website locally
to check the rendered documentation
and then push your changes to the repository as a pull request.

If you need help getting started,

> feel free to ask for help on the `#help-contributing` or `#keptn-docs` channels on the [Keptn Slack](https://keptn.sh/community/#slack).

Reviewer: Let's replace this with the CNCF #keptn channel since the keptn slack will be deleted soon.
```suggestion
feel free to ask for help on the `#keptn` channel on the [CNCF Slack](https://cloud-native.slack.com).
```

We were all new to this once and are happy to help you!

## Building the Documentation Locally

> To build and deploy the documentation in a docker container, execute

Reviewer: ```suggestion
To build and deploy the documentation in a container, execute
```

```shell
make docs-serve
```

> This will setup a docker container, install all needed dependencies,

Reviewer: ```suggestion
This will setup a container, install all needed dependencies,
```

build the documentation and serve it.

The URL on which your local documentation website is deployed will be
displayed in the logs.

> By default is should be `http://0.0.0.0:8000/`

Reviewer: ```suggestion
By default this should be `http://0.0.0.0:8000/`
```
